go.mod

```diff
@@ -103,6 +103,7 @@ require (
 	github.com/go-openapi/jsonpointer v0.19.5 // indirect
 	github.com/go-openapi/jsonreference v0.20.0 // indirect
 	github.com/go-openapi/swag v0.21.1 // indirect
+	github.com/go-sql-driver/mysql v1.6.0 // indirect
```

Comment:
any idea how we wound up with go.mod updates from this PR?

Reply:
I think it can from a rebase off main, I don't remember committing any go.mod until having to rebase

Reply:
go mod tidy after clearing cache still seems to have those entries

Reply:
ok, well, leave it for now i guess and once the PR is otherwise complete, we'll see if CI is blowing up around this. it properly checks for weirdness in go.mod